sos/report/plugins/openstack_tripleo.py

```diff
@@ -38,10 +40,7 @@ def postproc(self):
                   r'([":\s]+)(.*[^"])([",]+)'
         rgxp = re.compile(secrets, re.IGNORECASE)
 
-        self.do_path_regex_sub('/var/lib/tripleo-config/',
-                               rgxp, r'\1\3*********\5')
-        self.do_path_regex_sub('/etc/puppet/hieradata/',
-                               rgxp, r'\1\3*********\5')
-
+        for path in self.tripleo_log_paths:
+            self.do_path_regex_sub(path, rgxp, r'\1\3*********\5')
```

Comment:
This would significantly expand the reach of the obfuscation. Have you verified that this produces the desired effect on an actual OSP node?

Code-wise this looks fine.

Reply:
I have verified this is working on OSP16.1. That said if we think the increase in scope is too large I can scale this back.

Reply:
Scope is fine - just always like to verify we're not accidentally scrubbing unintentional data when we expand postprocessing.

Thanks!